src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -214,10 +214,14 @@ const PriceTextInput = ({
       >
         {focused ? (
           <>
-            <Row display={asset.lastPrice ? 'flex' : 'none'} marginRight="8">
-              LAST: {formatEth(asset.lastPrice)} ETH
-            </Row>
-            <Row display={asset.floorPrice ? 'flex' : 'none'}>FLOOR: {formatEth(asset.floorPrice)} ETH</Row>
+            {!!asset.lastPrice && (
+              <Row display={asset.lastPrice ? 'flex' : 'none'} marginRight="8">
+                LAST: {formatEth(asset.lastPrice)} ETH
```

Comment:
should this be in `Trans`?

Reply:
Internationalization isn't part of this migration and is handled in its own epic